tests/test_explanation.py

```diff
@@ -217,7 +210,7 @@ def test_explanation_with_token_granularities_oom_regression(sync_client: Client
 
 ### Response:"""
     target_text = " India won the Battle of Waterloo."
-    model_name = "luminous-base-control"
+    model_name = "luminous-base"
```

Comment:
Why move away from control?

Reply:
as it is (currently at least) not available in our new environment